roles/atomic_upgrade/tasks/main.yml

```diff
@@ -35,9 +35,8 @@
 - block:
 
     - name: Workaround BZ1290281
-      selinux:
-        policy: targeted
-        state: permissive
+      # Only temporary, don't touch /etc/sysconfig/selinux
```

Comment:
As long as you're touching these lines, how about (1) an explanation of the bug, something like "atomic host upgrade barfs with fsetxattr error", and (2) double-checking and maybe fixing the bz number, since I don't think that's the right one?